system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -374,6 +375,13 @@ class CLdapConfig : public CInterface, implements ILdapConfig
             LdapUtils::normalizeDn(dnbuf.str(), m_basedn.str(), m_filescope_basedn);
 
         dnbuf.clear();
+        cfg->getProp(".//@viewsBasedn", dnbuf);
+        if(dnbuf.length() == 0)//TODO REMOVE THIS WHEN CONFIGMGR SETS viewsBasedn
```

Comment:
My comment is a little misleading. If an existing config file is used, this property will not exist. I believe we need to leave it in, so I will make the comment more appropriate